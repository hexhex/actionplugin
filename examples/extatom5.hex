% -*- prolog -*-

in0(nil).
in2(c).
c(i).
c(j).

bar(X) :- &testB[IN,IN](X), in0(IN).
foo(X) :- &testB[IN,IN](X), in2(IN).
