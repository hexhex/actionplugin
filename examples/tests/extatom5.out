{foo(foo), bar(bar), in2(c), c(j), c(i), in0(nil)}
